spec/factories/rewards.rb

```diff
@@ -2,8 +2,8 @@
 
 FactoryBot.define do
   factory :reward do
-    sequence(:title) { |i| "Title#{i}" }
-    sequence(:description) { |i| "Description#{i}" }
+    sequence(:title) { |i| "Title#{i}abc" }
```

Comment:
why do you need this abc at the end?

Reply:
because "Title1" can be found within "Title15"

Reply:
so let's use Title01 insted ;) - check this out: `"%2d" % 1`